src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -587,6 +598,71 @@ private void testEditInstructorAction() throws Exception {
         courseEditPage.selectRoleForInstructor(editInstructorIndex, "Co-owner");
     }
     
+    private void testCancelEditInstructorAction() throws Exception {
+
+        ______TS("success: cancel editing an instructor role from Co-owner to Manager");
+        
+        int editInstructorIndex = 7;
+        
+        courseEditPage.clickEditInstructorLink(editInstructorIndex);
+        courseEditPage.selectRoleForInstructor(editInstructorIndex, "Manager");
+        courseEditPage.clickCancelEditInstructorLink(editInstructorIndex);
+        assertFalse(courseEditPage.isRoleSelectedForInstructor(editInstructorIndex, "Manager"));
+        assertTrue(courseEditPage.isRoleSelectedForInstructor(editInstructorIndex, "Co-owner"));
+        
+        ______TS("success: cancel editing an instructor role from Co-owner to Custom");
+        
+        courseEditPage.clickEditInstructorLink(editInstructorIndex);
+        courseEditPage.selectRoleForInstructor(editInstructorIndex, "Custom");
+        courseEditPage.clickCourseLevelPrivilegesLink(editInstructorIndex, 1);
+        courseEditPage.clickAddSectionLevelPrivilegesLink(editInstructorIndex);
+        courseEditPage.clickSectionSelectionCheckBox(editInstructorIndex, 0, 0);
+        courseEditPage.clickSectionLevelPrivilegeLink(editInstructorIndex, 0, 0);
+        courseEditPage.clickSessionLevelInSectionLevel(editInstructorIndex, 0);
+        courseEditPage.clickAddSectionLevelPrivilegesLink(editInstructorIndex);
+        courseEditPage.clickSectionSelectionCheckBox(editInstructorIndex, 1, 1);
+        courseEditPage.clickSectionLevelPrivilegeLink(editInstructorIndex, 1, 0);
+        courseEditPage.clickAddSectionLevelPrivilegesLink(editInstructorIndex);
+        courseEditPage.clickSectionSelectionCheckBox(editInstructorIndex, 2, 2);
+        courseEditPage.clickSectionLevelPrivilegeLink(editInstructorIndex, 2, 0);
+        courseEditPage.clickCancelEditInstructorLink(editInstructorIndex);
```

Comment:
The code here follows the style of other tests in the class, which has already been partially fixed in #5732; merging that branch in and changing the style here to match might help. 

Code such as `courseEditPage.clickSectionSelectionCheckBox(editInstructorIndex, 0, 0);` will remain as it is however, since there is no special meaning that is not derived directly from the numbers ("click the checkbox representing the first section in the first section group panel" (due to 0-based index) in this case).

Other more problematic code (testing feedback response comments, with a series of 4 integers as an identifier) comes to mind, but similarly to this case, there is no additional meaning that can be derived from the numbers.